src/main/java/fiveavian/proxvc/vc/AudioInputDevice.java

```diff
@@ -47,6 +44,15 @@ public synchronized ByteBuffer pollSamples() {
         return samples;
     }
 
+    public static synchronized String[] getSpecifiers(){
```

Comment:
`synchonized` was used to ensure only 1 thread could access this class's methods at a time so that the device isn't read while reopening. it doesn't make sense for the static method since it doesn't rely on any state apart from OpenAL initialization.